OpenSim/Actuators/Millard2012AccelerationMuscle.cpp

```diff
@@ -198,7 +198,7 @@ void Millard2012AccelerationMuscle::buildMuscle()
     fcphi.setName(tmp);
 
      //Ensure all sub objects are up to date with properties;
-    actMdl.ensureModelUpToDate();
+    actMdl.finalizeFromProperties();
```

Comment:
Will this be changed eventually? If not, maybe there can be a comment explaining this apparent inconsistency.